wikicurses/main.py

```diff
@@ -35,6 +37,111 @@ def tabComplete(text, matches):
     return match
 
 
+def showImages():
+    """
+    Launch program to display the images for the current article.
+
+    The program is specified in the config, along with it's args.
+    Otherwise, the default is `feh`.
+    """
+    if 'wikipedia.org' not in wiki.siteurl:
+        ex.notify('Cancelled: Only works on Wikipedia')
+        return
```

Comment:
AFAIK vimwiki does not run on MediaWiki. It may also not work on some older (and future) versions of MediaWiki. To increase compatibility, I think you should work with the [API endpoint](https://www.mediawiki.org/wiki/API:Main_page) instead of [index.php](https://www.mediawiki.org/wiki/Special:MyLanguage/Manual:Parameters_to_index.php) - it provides an [error reporting](https://www.mediawiki.org/wiki/Special:MyLanguage/API:Errors_and_warnings) mechanism that you could use to produce meaningful error messages. It is also much more extensively documented and obviously has more features.